images/miq-app/docker-assets/container-scripts/container-deploy-common.sh

```diff
@@ -113,6 +116,29 @@ fi
 
 }
 
+function check_if_new_replica() {
+# Description
+# Check for pre-existing server data into PV, if not found, we assume a new server/replica case
+# Always skip if we are performing a new_deployment
+
+echo "== Checking for existing data on server PV =="
+
+if [[ ! -d ${PV_CONTAINER_DATA_DIR} && ${DEPLOYMENT_STATUS} != new_deployment ]]; then
```

Comment:
The initialization logs for the server-data directory would have never happened on a new_deployment case or new_replica. They happen afterwards see here : 

https://github.com/ManageIQ/manageiq-pods/blob/master/images/miq-app/docker-assets/appliance-initialize.sh#L41

Keep in mind appliance-initialize.sh dictates the order of events, check_deployment_status is just a function before rest of actions happen.

Also, please don't confuse with the server-deploy logs, which are a different directory.